CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Internal
 * Cleaned up the console output produced by `./tests`. ([#2548](https://github.com/realm/realm-js/pull/2548))
+* Updated to Realm Sync v4.7.10.
```

Comment:
```suggestion
* Updated to Realm Sync from 4.7.8 to 4.7.10.
```

Reply:
Are there no public changes that should be mentioned?

Reply:
It is probably worth mentioning that changeset scanning for synced Realm is faster.

Reply:
Btw, is your suggestion above a new general "rule" for changelogs?